signale.js

```diff
@@ -257,6 +257,19 @@ class Signale {
     this._log(this._buildSignale(this._types[type], ...messageObj), this._types[type].stream);
   }
 
+  _padSpace(str, targetLength) {
```

Comment:
The method can be still named `_padEnd`, so its functionality can be as obvious as possible;

```js
  _padEnd(str, targetLength) 
```